src/common/utility_unix.cpp

```diff
@@ -111,6 +111,10 @@ void Utility::setLaunchOnStartup(const QString &appName, const QString &guiName,
     }
 }
 
+void Utility::migrateLaunchOnStartup()
```

Comment:
Do we need to make this function publically available? 
If not we won't need to implement it on the other platfomrs.

Reply:
Hmm, I'll check if I can put this in the `Platform` start-up code.

Reply:
The best I can do is to call the code from `Platform::create()` (that would hide it for non-mac platforms), and call that as follows:
```c++
auto platform = Platform::create(APPLICATION_REV_DOMAIN);
```
That is because the migration code is in `src/common`, and doesn't have the app domain available. 

What's your opinion?

Reply:
Sounds like a good solution.

Reply:
@TheOneRing fixed